actions/get_formatted_error.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# Copyright 2019 Encore Technologies
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+from six import string_types
+from lib.base_action import BaseAction
+
+
+IGNORED_ERROR_TASKS = [
```

Comment:
These would be encore specific ignored tasks. We need to make this an input and pass it to the function where it is used